site/handler.go

```diff
@@ -146,16 +146,19 @@ func (h *handler) serveIndex(w http.ResponseWriter, r *http.Request) {
 
 func (h *handler) serveChart(w http.ResponseWriter, r *http.Request, path string) {
 	root := "https://storage.googleapis.com/open-match-chart/chart"
+	h.withCors(w)
```

Comment:
Just curious why are we adding cors rules here if we already have CORS rules setup in our cloud storage bucket

Reply:
This handles http://open-match.dev/ The GCS CORS is from http://storage.googleapis.com/